cafebabel/templates/articles/macros.html

```diff
@@ -82,10 +82,10 @@ <h4>Translation by:</h4>
   <img src={{ url_for('static', filename='img/article-thumbnail-default.jpg') }} alt="Default thumbnail picture">
 {% endmacro %}
 
-{% macro card(article) %}
+{% macro card(article, image_dimensions='800x550') %}
```

Comment:
Actually it would be 750x290 except for homepage editor's pick article (the one on top) : 1200x467.

Reply:
Oh you are right, on the homepage it's not `max-width: 75rem`, it's 120rem * 4/6 = 793. Sorry my bad !